tests/src/buildApplication/generateFunctionsMap.test.ts

```diff
@@ -171,9 +171,19 @@ describe('generateFunctionsMap', async () => {
 			});
 		});
 
-		test('succeeds for prerendered favicon', async () => {
+		test('succeeds for prerendered favicon + json', async () => {
```

Comment:
I think it would be nicer to have two different separated tests, I don't see any real reason/benefit for having these two things tested together (it makes the test just more complex and it can fail if either favicon or json handling is broken without clearly indicating which one actually is)

WDYK?

Reply:
sure i'll put them separate, idm